app/services/db-export/send-to-s3-bucket.service.js

```diff
@@ -0,0 +1,63 @@
+'use strict'
+
+/**
+ * Sends a file to our AWS S3 bucket
+ * @module SendToS3BucketService
+ */
+
+const { S3Client, PutObjectCommand } = require('@aws-sdk/client-s3')
+const fs = require('fs')
+const S3Config = require('../../../config/s3.config')
+const path = require('path')
```

Comment:
Only a minor thing but this doesn't quite meet our [team convention](https://github.com/DEFRA/water-abstraction-team/blob/main/coding_conventions.md#top-of-js-files)
```suggestion
const fs = require('fs')
const path = require('path')
const { PutObjectCommand, S3Client } = require('@aws-sdk/client-s3')

const S3Config = require('../../../config/s3.config')
```